module1/homework/lengthSort/sort.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <deque>
+#include <forward_list>
+#include <string>
+
+bool compare(const std::string& first, const std::string& second);
+std::deque<std::string> lengthSort(std::forward_list<std::string>& flist);
```

Comment:
Isn't "flist" a typo?

Reply:
It's a name of an argument for lengthSort function (it's called flist, because its type is forward_list, a letter f came from a word foward)

Reply:
Sorry! Never mind!